test/micro/org/openjdk/bench/java/lang/StringBuilders.java

```diff
@@ -398,6 +404,7 @@ public void setup() {
         }
 
         private void generateData() {
+            sb = new StringBuilder(length);
```

Comment:
This looks weird, given there is a `sb` initialization two lines below. Is this `sbEmpty`, really?

Reply:
Yes, updating the name to avoid the confusing shadowing.